feat: add multilingual navbar

Guileas opened the pull request:
Fixes: https://github.com/RatanShreshtha/DeepThought/issues/11

williamdes commented on README.md:
```diff
@@ -21,9 +21,33 @@ Go into your sites directory, and type `zola serve`. You should see your new sit
 
-## Theme Options
+## Multilingual Navbar
+**NOTE** : `navbar_items` store in the `config.toml` file will be use as the default navbar. If you don't have any other language or if you forgot to create a navbar for a specific language your users will always see this navbar.
```
```suggestion
**NOTE** : `navbar_items` stored in the `config.toml` file will be used as the default navbar. If you don't have any other language or if you forgot to create a navbar for a specific language your users will always see this navbar.
```

RatanShreshtha commented:
@Guileas Add few screenshots as well, rest all looks good to me.

Guileas commented:
@RatanShreshtha I've modified my PR to fix some issues I had for multiple translations. 
I've added the language code to the `navbar_items` array to make everything work but no more default navbar if the user add a language code and forgot to add a translation in the `navbar_items` array for this language.
